### ISSUE TITLE
Export the lua functions underlying the commands

### ISSUE THREAD

**Issue by tmke8** [neovim/nvim-lspconfig#2521]
### Language server

all

### Requested feature

The pyright config exports the `PyrightOrganizeImports` command, which I previously had bound to a key mapping like this:

```lua
vim.api.nvim_set_keymap('n', '<leader>i', '<cmd>PyrightOrganizeImports<cr>', {noremap = true})
```

However, I would like to replace that with `vim.keymap.set` for which it would be most efficient to directly reference the corresponding lua function:

https://github.com/neovim/nvim-lspconfig/blob/0f94c5fded29c0024254259f3d8a0284bfb507ea/lua/lspconfig/server_configurations/pyright.lua#L19-L25

So that I can write something like
```lua
vim.keymap.set('n', '<leader>i', organize_imports)
```

However, this `organize_imports()` function is not exported, so I can't do that. My current work-around is this:

```lua
vim.keymap.set('n', '<leader>i', function()
  vim.lsp.buf.execute_command({ command = 'pyright.organizeimports', arguments = { vim.uri_from_bufnr(0) } })
end, { buffer=true })
```
Basically, I just copied the definition of the function into my config file.

**Comment by glepnir** [comment]
use command and function is same.
